### DISCUSSION TITLE
You need a serial terminal

### DISCUSSION THREAD

**Discussion by mmiscool** [DustinBrett/daedalOS#153] [Feature Requests]
Love what you are doing. 
This was the first time I was able to actually make sense of what boxed wine does.

A shameless plug for a project I did a little while back. https://serialterminal.com
It uses the web serial API and works in chrome and edge. 

Source code here. Might be good to add to your web desktop. 
https://github.com/mmiscool/serialTerminal.com

**Comment by DustinBrett** [comment]
Thanks! This looks very cool and I hadn't thought of doing serial stuff but will for sure add this to my to do list. Thanks for the idea!
